docs/ingress-gateway-asm-manifests/base/pdb-v1.yaml

```diff
@@ -0,0 +1,13 @@
+# Optional: PodDisruptionBudget will instruct Kubernetes to ensure at least 1 replica
+# is available when possible
```

Comment:
**Praise:** Thanks for including at explanation of each file at the top. This will help our users.